src/goto-programs/goto_program.h

```diff
@@ -82,6 +82,13 @@ inline bool operator<(
   return order_const_target<codet, exprt>(i1, i2);
 }
 
+inline bool operator<(
```

Comment:
This duplicates https://github.com/diffblue/cbmc/pull/1247, but I needed it in order to build - once 1247 is merged I'll remove this